pkg/pillar/types/global.go

```diff
@@ -689,25 +696,54 @@ func NewConfigItemSpecMap() ConfigItemSpecMap {
 	configItemSpecMap.GlobalSettings = make(map[GlobalSettingKey]ConfigItemSpec)
 	configItemSpecMap.AgentSettings = make(map[AgentSettingKey]ConfigItemSpec)
 
-	configItemSpecMap.AddIntItem(ConfigInterval, 60, 5, 0xFFFFFFFF)
-	configItemSpecMap.AddIntItem(MetricInterval, 60, 5, 0xFFFFFFFF)
-	configItemSpecMap.AddIntItem(ResetIfCloudGoneTime, 7*24*3600, 120, 0xFFFFFFFF)
+	// timer.config.interval(seconds)
+	// MaxValue needs to be limited. If configured too high, the device will wait
+	// too long to get next config and is practically unreachable for any config
+	// changes or reboot through cloud.
+	configItemSpecMap.AddIntItem(ConfigInterval, 60, 5, HourInSec)
+	// timer.metric.interval (seconds)
+	// Need to be careful about max value. Cloud uses this to indicate if a
+	// device is online or suspect state.
```

Comment:
And some controller will declare it as suspect after 3 minutes, so that clearly isn't the motivation for the max of 1 hour. So please update the comment.

BTW: We tend to use "controller" to refer to the controller in EVE code. Some documents alternatively refer to it as EVC. But "cloud" is not something we use in new code/comments.

Reply:
"so that clearly isn't the motivation for the max of 1 hour."
--- Thats not the motivation.. Intent is to say - keep it in sync with what Controller expects or the device can show up as suspect. Let me fix the working..

Yes.. Controller is a better term.. Will fix it.